pkg/controllers/cloud/cloudentityselector_controller.go

```diff
@@ -148,116 +198,31 @@ func (r *CloudEntitySelectorReconciler) processCreateOrUpdate(selector *cloudv1a
 		return err
 	}
 
-	if !preExists {
-		go wait.Until(accPoller.doAccountPoller, time.Duration(accPoller.pollIntvInSeconds)*time.Second, accPoller.ch)
-	}
 	return nil
 }
 
 func (r *CloudEntitySelectorReconciler) processDelete(selectorNamespacedName *types.NamespacedName) error {
-	poller := r.removeAccountPoller(selectorNamespacedName)
-	if poller == nil {
-		return nil
-	}
-	cloudInterface, err := cloudprovider.GetCloudInterface(common.ProviderType(poller.cloudType))
-	if err != nil {
+	accountNamespacedName, found := r.selectorToAccountMap[*selectorNamespacedName]
+	if !found {
+		err := fmt.Errorf("selector to Account mapping doesn't exist, selector %v", selectorNamespacedName.String())
+		r.Log.Error(err, "processDelete Failed", "selector", selectorNamespacedName.String())
```

Comment:
Please change the log message.
Here selector will be output twice(once from L207 and other from L208)